airflow/www_rbac/views.py

```diff
@@ -1062,6 +1062,7 @@ def trigger(self, session=None):
                     conf=conf
                 )
 
+        dag = dagbag.get_dag(dag_id)
```

Comment:
Lol what? Do we have _no_ tests of this?

Reply:
This only errors if the Webserver does not have access to Dag Files (like was mentioned in the issue when the DAGs dir was not mounted to Webserver).

We might have tests to see when the config is enabled this still works but we might not have tests to check the Webserver doesn't have access to Dag file.

Have kept the PR in draft so that I can add some tests.

Reply:
@ashb I have added the tests to cover this case.

Reply:
I will add these tests to Master too once this PR is merged